contracts/DepegProduct.sol

```diff
@@ -87,16 +87,19 @@ contract DepegProduct is
 
 
     function applyForPolicy(
+        address wallet,
```

Comment:
`protectedWallet` ?

Reply:
prefer the shorter name as there is only one wallet (which is the protected wallet)